GitHub CI Build Failing

For some reason, GitHub Action's CI is failing. I had verified with NevolutionX repository is building with current nxdk state. Plus force push xkts's master branch in my own fork with current nxdk state lead to failure to build.

The problem:
```
[ CC       ] /home/runner/work/xbox_kernel_test_suite/xbox_kernel_test_suite/projects/xbox_kernel_test_suite/irt_tests.obj
[ CC       ] /home/runner/work/xbox_kernel_test_suite/xbox_kernel_test_suite/projects/xbox_kernel_test_suite/av_tests.obj
make: nxdk-cc: Command not found
make: nxdk-cc: Command not found
make: *** [/home/runner/work/xbox_kernel_test_suite/xbox_kernel_test_suite/projects/xbox_kernel_test_suite/../../Makefile:136: /home/runner/work/xbox_kernel_test_suite/xbox_kernel_test_suite/projects/xbox_kernel_test_suite/irt_tests.obj] Error 127
make: *** Waiting for unfinished jobs....
make: *** [/home/runner/work/xbox_kernel_test_suite/xbox_kernel_test_suite/projects/xbox_kernel_test_suite/../../Makefile:136: /home/runner/work/xbox_kernel_test_suite/xbox_kernel_test_suite/projects/xbox_kernel_test_suite/av_tests.obj] Error 127
Error: Process completed with exit code 2.
```

The origin of cause is unknown.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -37,7 +37,9 @@ jobs:
           sudo apt-get -y install lld llvm
       - name: Build
         working-directory: ${{ steps.xkts.outputs.dir }}
-        run: make -j$(nproc)
+        run: |
+          eval $(../../bin/activate -s) # setup nxdk variables into environment
+          make -j$(nproc)
       - name: Set artifact + release parameters
         id: parameters
         run: |

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,3 @@
 XBE_TITLE=kernel_test_suite
 SRCS = $(wildcard $(CURDIR)/*.c)
-NXDK_DIR = $(CURDIR)/../..
 include $(NXDK_DIR)/Makefile

